nautobot/docs/additional-features/job-hooks.md

```diff
@@ -0,0 +1,52 @@
+# Job hooks
+
+A job hook is a mechanism for automatically starting a [job](./jobs.md) when an object is changed. Job hooks are similar to [webhooks](../models/extras/webhook.md) except change events initiate a `JobHookReceiver` job instead of a web request. Job hooks are configured in the web UI under Jobs > Job Hooks.
+
+## Job hook receivers
+
+Job hooks are only able to initiate a specific type of job called a job hook receiver. These are jobs that subclass the `nautobot.extras.jobs.JobHookReceiver` class. Job hook receivers are similar to normal jobs except they are hard coded to accept only an `object_change` [variable](jobs.md#variables). Job hook receivers are hidden from the jobs listing UI by default but function similarly to other jobs. The job hook receiver class only implements one method called `receive_jobhook`.
```

Comment:
```suggestion
Job Hooks are only able to initiate a specific type of job called a **Job Hook Receiver**. These are jobs that subclass the `nautobot.extras.jobs.JobHookReceiver` class. Job hook receivers are similar to normal jobs except they are hard coded to accept only an `object_change` [variable](jobs.md#variables). Job Hook Receivers are hidden from the jobs listing UI by default but otherwise function similarly to other jobs. The `JobHookReceiver` class only implements one method called `receive_jobhook`.
```